dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/MetadataDumperTest.java

```diff
@@ -25,10 +33,123 @@
 @RunWith(JUnit4.class)
 public class MetadataDumperTest {
 
+  private File file;
+  private MetadataDumper dumper = new MetadataDumper().withExitOnError(false);
+  Connector connector = new HiveMetadataConnector();
+
+  @After
+  public void tearDown() throws IOException {
+    if (file == null || !file.exists()) return;
+    if (file.isDirectory()) FileUtils.deleteDirectory(file);
+    else file.delete();
+  }
+
   @Test
   public void testInstantiate() throws Exception {
-    MetadataDumper dumper = new MetadataDumper();
-    // dumper.run("--help");    // Calls System.exit(1)
+    dumper = new MetadataDumper();
     dumper.run("--connector", new BigQueryLogsConnector().getName(), "--dry-run");
   }
+
+  @Test
+  public void testCreatesDefaultOutputZip() throws Exception {
+    // Arrange
+    String name = connector.getDefaultFileName(false);
+    file = new File(name);
+
+    // Act
+    dumper.run("--connector", connector.getName());
+
+    // Assert
+    Assert.assertTrue(file.exists());
+  }
+
+  @Test
+  public void testCreatesDefaultOutputZipInProvidedDirectory() throws Exception {
+    // Arrange
+    String name = connector.getDefaultFileName(false);
```

Comment:
I would suggest to use the literal `dwh-migration-hiveql-metadata.zip` as you do in other tests below. The literal is more explicit and this also avoids logic in tests.

Similarly in the test above.